src/Textual/Textual.jl

```diff
@@ -17,18 +17,43 @@ using ..FastAI.Datasets
 
 using ..FastAI.Datasets
 
+# for tests
+using ..FastAI: testencoding
+
+# extending
+import ..FastAI:
+    blockmodel, blockbackbone, blocklossfn, encode, decode, checkblock,
+    encodedblock, decodedblock, showblock!, mockblock, setup, encodestate,
+    decodestate
+
+
+
 import Requires: @require
 
 using InlineTest
 using Random
+using TextAnalysis
+using DataStructures
+
+using WordTokenizers: TokenBuffer, isdone, character, spaces, nltk_url1, nltk_url2, nltk_phonenumbers
 
 include("recipes.jl")
 include("blocks/text.jl")
 include("transform.jl")
+include("encodings/textpreprocessing.jl")
+
+const _tasks = Dict{String,Any}()
+include("tasks/classification.jl")
+
 
 function __init__()
     _registerrecipes()
+    foreach(values(_tasks)) do t
+        if !haskey(FastAI.learningtasks(), t.id)
+            push!(FastAI.learningtasks(), t)
+        end
+    end
 end
 
-export Paragraph
+export Paragraph, TokenVector, TextClassficationSingle, Sanitize, Tokenize, replace_all_caps, replace_sentence_case, convert_lowercase
```

Comment:
I think we shouldn't export the replace_* and convert_* functions for now as the high-level exported interface should be used via the encodings or tasks.